app.js

```diff
@@ -1,30 +1,34 @@
 const express = require("express");
 
-
-const { getAllTopics, getAllEndpoints, getAllArticles } = require("./controllers.controllers");
-=======
-
+const {
+  getAllTopics,
+  getAllEndpoints,
+  getAllArticles,
+  getArticleById,
+  getCommentsByArticleId,
+} = require("./controllers.controllers");
 
 const app = express();
 app.use(express.json());
 
-app.get("/api/topics", getAllTopics );
+app.get("/api/topics", getAllTopics);
 app.get("/api/", getAllEndpoints);
+app.get("/api/articles", getAllArticles);
+app.get("/api/articles/:article_id", getArticleById);
+app.get("/api/articles/:article_id/comments", getCommentsByArticleId);
 
-app.get("/api/articles", getAllArticles)
-
-app.get("/api/articles/:article_id", getArticleById)
-
+app.use("*", (req, res, next) => {
+  res.status(404).send({ msg: "Endpoint not found :(" });
+});
 
 app.use((err, req, res, next) => {
     if (err.status && err.msg) {
-        res.status(err.status).send({ msg: err.msg });
+      res.status(err.status).send({ msg: err.msg });
+    } else if (err.code === "22P02") {
+      res.status(400).send({ msg: "Bad request" });
+    } else {
```

Comment:
I would add a console.log in your 500 error handler so that you know what the error coming back is - then you can handle it accordingly